app/components/EventListCompact/index.tsx

```diff
@@ -1,41 +1,42 @@
+import EmptyState from 'app/components/EmptyState';
+import EventItem, { type EventStyle } from 'app/components/EventItem';
+import { Flex } from 'app/components/Layout';
+import LoadingIndicator from 'app/components/LoadingIndicator';
 import type { Event } from 'app/models';
-import EmptyState from '../EmptyState';
-import EventItem, { type EventStyle } from '../EventItem';
-import { Flex } from '../Layout';
-import styles from './EventListCompact.css';
 
 type Props = {
   events: Array<Event>;
   noEventsMessage: string;
-  loggedIn: boolean;
   eventStyle?: EventStyle;
+  loading?: boolean;
+  classNameErrorMessage?: string;
```

Comment:
the `classNameErrorMessage` is used to style the `noEventsMessage`. Might be something i missed, but i think something like `classNameNoEventsMessage` makes the purpose of the field more clear.